ghost/i18n/locales/ar/comments.json

```diff
@@ -1,6 +1,5 @@
-{
-    "{{amount}} characters left": "{{amount}} حرف متبقي",
-    "{{amount}} comments": "{{amount}} تعليقات",
+    "{{amount}} characters left": "الحروف المتبقية: {{amount}}",
```

Comment:
You have lost the { and } around this file.  These are required for the i18next to understand your work.  Please correct.